### ISSUE TITLE
Preserve parameters when fine-tuning on a target dataset that has overlapping classes with source dataset

### ISSUE THREAD

**Issue by j-rausch** [facebookresearch/detectron2#2063]
Hi, I'm wondering whether it is possible to preserve some of the class-specific parameters when fine-tuning on a new target dataset.

For example:
-Initialize COCO-pretrained Mask R-CNN model (C=80).
-Fine-tune on a custom dataset with C=70 classes.
	-Of those 70 classes, 60 classes are identical to existing COCO classes

The default behaviour seems to be to entirely skip affected layers, e.g.:
`WARNING [09/23 11:28:03 fvcore.common.checkpoint]: 'roi_heads.box_predictor.cls_score.weight' has shape (81, 1024) in the checkpoint but (71, 1024) in the model! Skipped.`

Instead of re-initializing the parameters of e.g. 'roi_heads.box_predictor.cls_score' entirely, would it be possible to only randomly initialize parameters in the affected layers for new classes and preserve weights for existing classes (e.g. based on category name or by providing a source-to-target ID mapping)?

Thanks!

**Comment by ppwwyyxx** [comment]
You can [edit the checkpoint](https://detectron2.readthedocs.io/tutorials/models.html#load-save-a-checkpoint) so the weights can be initialized to what you want (e.g., using the category names to map them).
